src/jwt.ts

```diff
@@ -0,0 +1,15 @@
+import * as jwt from 'jsonwebtoken'
```

Comment:
can we remove this file? Otherwise we'll have it twice. I will move the one we already have to the src folder where it should be. sorry bout that!!

Reply:
Perfect. We can remove from route and place in src folder.